domain/src/main/kotlin/no/nav/su/se/bakover/domain/Copyable.kt

```diff
@@ -11,4 +11,16 @@ sealed class CopyArgs {
         data class NyPeriode(val periode: Periode) : Tidslinje()
         object Full : Tidslinje()
     }
+
+    data class BegrensetTil(val periode: Periode) : CopyArgs() {
```

Comment:
Kunne gjerne hatt en javadoc her for å forklare litt mer. Klassenavnet og funksjonsnavnet bør vel være navngitt som motsetninger? Jeg ville tenkt at dersom `A.copy(BegrensetTil(2021)), så må A være innenfor 2021 (sånn som den er implementert nå, må den dekke hele perioden man er begrensen av), hvis ikke får man null? Da kunne kanskje funksjonen het noe i retningen begrensHvisUtenfor? :shrug: